test-wrapper/src/main/java/pl/symentis/entities/jcstress/JCStressTest.java

```diff
@@ -6,7 +6,7 @@
 @Entity("jcstress-tests")
 public record JCStressTest(
     @Id
-    JCStressTestId testId,
+    String requestId,
```

Comment:
why? you still keep JmhBenchmarkId and removed this type? I think it would be more consisten

Reply:
We used to have a commitSha and run attempt as a id parts - it's not relevant any more since we're running test not only from CI/CD